packages/expo-updates/android/src/main/cpp/BSPatchModule.cpp

```diff
@@ -0,0 +1,40 @@
+#include "BSPatch.h"
+
+namespace jni = facebook::jni;
+
+extern "C" int main(int argc, char *argv[]);
```

Comment:
not a fan to call the main here. i recommend to write our entry point. at least just rename main to some other symbol.
`main` has its special meaning to programs and a platform may only allow single C convention main in the linked unit